Today/Controllers/ReminderDetailViewController/ReminderDetailEditDataSource.swift

```diff
@@ -59,13 +59,14 @@ class ReminderDetailEditDataSource: NSObject {
             if indexPath.row == 0 {
                 cell.textLabel?.text = formatter.string(from: reminder.dueDate)
             } else {
-                if let dueDateCell = cell as? EditDateCell {
-                    dueDateCell.configure(dueDate: reminder.dueDate) { date in
-                        self.reminder.dueDate = date
-                        self.reminderChangeAction?(self.reminder)
-                        let indexPath = IndexPath(row: 0, section: section.rawValue)
-                        tableView.reloadRows(at: [indexPath], with: .automatic)
-                    }
+                guard let dueDateCell = cell as? EditDateCell else {
```

Comment:
Always be consistent. Code should be consistent in every aspect. Handle every case with best and single approach.

Reply:
We are following the practice of assigning  storyboard identifiers to VIewController, TableViewCell, CollectionViewCell etc(any storybaordID that need to be used to initialize object) as the class name for this element and put in a computed property in its UIKit base class to get the Identifier from the class.